cmd/api/src/api/signature.go

```diff
@@ -34,48 +34,6 @@ import (
 
 const ErrTemplateHMACSignature string = "unable to compute hmac signature: %w"
 
-// tee takes a source reader and two writers. The function reads from the source until exhaustion. Each read is written
```

Comment:
Good catch on the tee not being used anywhere beyond old tests. This is excellent cleanup.